connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaPropertyTransformer.java

```diff
@@ -129,11 +129,11 @@ private static void setValue(
       } else {
         String value = (String) consumerRecord.value();
         kafkaInboundMessage.setRawValue(value);
-        var json = StringEscapeUtils.unescapeJson(value);
-        kafkaInboundMessage.setValue(objectReader.readTree(json));
+        kafkaInboundMessage.setValue(objectReader.readTree(value));
       }
     } catch (Exception e) {
-      LOG.debug("Cannot parse value to json object -> use the raw value");
+      LOG.error(e.getMessage());
+      LOG.error("Cannot parse value to json object -> use the raw value");
```

Comment:
Is it an error? Something we want to act on?

Reply:
Good question! The reason behind this is to be able to handle the case when we process a malformed message. The raw value (string value) will still be available, even though we cannot deserialize the message.

Reply:
Ok, so you changed the log to an error one so that we can spot these kinds of issues in a GCP log for instance 👌 

I don't know if there's a reason for the `LOG.error(e.getMessage());` but I think you can use:

`LOG.error("Cannot parse value to json object -> use the raw value", e);`

Reply:
Good idea, adjusted it.